cmd/minikube/cmd/start_flags.go

```diff
@@ -281,8 +281,8 @@ func initNetworkingFlags() {
 	startCmd.Flags().Int(sshSSHPort, defaultSSHPort, "SSH port (ssh driver only)")
 
 	// socket vmnet
-	startCmd.Flags().String(socketVMnetClientPath, "/opt/socket_vmnet/bin/socket_vmnet_client", "Path to the socket vmnet client binary")
-	startCmd.Flags().String(socketVMnetPath, "/var/run/socket_vmnet", "Path to socket vmnet binary")
+	startCmd.Flags().String(socketVMnetClientPath, "", "Path to the socket vmnet client binary")
```

Comment:
(lets add QEMU only to the beggining of this two flags, so ppl dont read it when not relevant to them)


```Qemu driver only:....
```

Reply:
We have `(Docker driver only)` at the end of a lot of our flags, think I should follow that convention to make it consistent

Reply:
sure that works, and later we can decide to change the style